Private/Invoke-EventLogVerification.ps1

```diff
@@ -1,10 +1,8 @@
 function Invoke-EventLogVerification ($Results, $Dates) {
-
     $Logs = @()
     foreach ($result in $Results) {
-
         if ($result.EventOldest -gt $Dates.DateFrom) {
-            Write-Color @script:WriteParameters '[W] ', 'Warning: ', $result.LogName, ' log on ', $result.Server, " doesn't cover whole date range requested. Oldest event ", $result.EventOldest, ' while requested ', $Dates.DateFrom, '.' -Color Blue, White, Yellow, White, Yellow, White, Yellow, White, Yellow
+            $Logger.AddWarningRecord("$($result.LogName) log on $($result.Server) doesn't cover whole date range requested. Oldest event $($result.EventOldest) while requested $($Dates.DateFrom)")
             $Logs += "<strong>$($result.LogName)</strong> log on <strong>$($result.Server)</strong> doesn't cover whole date range requested. Oldest event <strong>$($result.EventOldest)</strong> while requested <strong>$($Dates.DateFrom)</strong>."
```

Comment:
This is mainly for visibility for you. For future we should see if we can automate Logger for Html as well. Maybe we should add those warning,error, information logs as part of HTML message with a report so that admin can review if the process went good or had errors in it. It's easier to review 1 log from 1 job then 50 jobs every few weeks. Should be an option  in config for that.

Reply:
I think the same.
But I’m not sure which option to choose: attach the entire log to the letter, add html errors and warnings to the body of the letter, or maybe both.

Reply:
I would say it needs a bit nicer formatting in HTML output. Attaching error log may be nice too, as an option. Also a summary on top of errors, warning, information (for quick assessment) would be nice addition.